Do not install build-dep, build xfce components in correct order

Skunnyk opened the pull request:
- Do not install "build-dep", because they also provide debian package
  for example libexo, libxfce4ui, libxfce4util, libxfce4panel etc,
which can lead to confusion and bad linking
- Build Xfce in correct order, so we can start "from scratch"

schuellerf commented:
despite the typo in Dockerfile:24 it does not build ... I'll check later why - but I'm looking forward to removing this package mess :-)
Thanks for your support so far

Skunnyk commented:
Silly me. Let me double check. 
Also, not sure all theses gstreamer packages are really necessary !

Skunnyk commented:
Updated with fixed syntax. Container builds and start fine here

I added a `make clean` in the loop to reduce container size a bit
```
schuellerf/xfce-test   latest        1a0a033e50d6        About a minute ago   2.97GB
```
